### ISSUE TITLE
"Use this folder" button not visible at first

### ISSUE THREAD

**Issue by szpak** [Tommy-Geenexus/exif-eraser#20]
## Steps to reproduce:
1. Settings -> Default Save Path (or anything else what shows that "dialog")

## Actual behavior:
Unable to choose current folder - no "Use this folder" button on the bottom. It is needed to step up and step down into the folder again to see that button.

## Expected behavior:
The is that button and can be used from the first time.

## Additional information:
Android 11, 4.0.0 from Play Store. It worked fine with 3.x.

**Comment by Tommy-Geenexus** [comment]
Hi, thanks for your report.
This is not an issue with this app, but rather with the default file manager on Android (since Android 11).
I don't know whether it's a bug or intended behaviour on Google's part.

Did you test version 3.x with Android 10 or 11?
Just to make sure i tested 3.x and managed to reproduce the above behaviour.

**Comment by szpak** [comment]
I've been using 3.x since the beginning, but I believe it was Android 10 then. I've bumped into that as 4.0 seems to restart saved default path and I had to do it again (which exposed the problem).

I reported it as I though there is "some flag to be set" which is missing, however, that might be indeed the behavior of the default file browser in Android 11. In that case it might be reasonable to close it with "3rd-party issue), especially that there is am existing workaround. I will report back once I meet any other app that use that file selector.
